docs/templates/github-actions/scan-and-fix/frogbot-scan-and-fix-npm.yml

```diff
@@ -1,11 +1,5 @@
 name: "Frogbot Scan and Fix"
 on:
-  push:
-    # Creating fix pull requests will be triggered by any push to one of these branches.
-    # You can add or replace to any branch you want to open fix pull requests for.
-    branches:
-      - "main"
-      - "master"
   schedule:
     # The job will run once a day at 00:00 GMT.
```

Comment:
The job will run once a day at 00:00 GMT.
-->
The repository will be scanned once a day at 00:00 GMT.

(let's change in all templates).